cranelift/native/src/lib.rs

```diff
@@ -167,7 +167,7 @@ mod tests {
 
             if cfg!(all(target_os = "macos", target_arch = "aarch64")) {
                 assert_eq!(isa.default_call_conv(), CallConv::AppleAarch64);
-            } else if cfg!(any(unix, target_os = "nebulet")) {
+            } else if cfg!(unix) {
```

Comment:
This change is because rustc now warns that `target_os = "nebulet"` doesn't exist. I think that means the target was probably removed here. I don't know what nebulet is myself, though, so I'm just obeying rustc.